tests/Screens.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import HomeScreen from '../src/home/HomeScreen';
+import GameScreen from '../src/game/GameScreen';
+import GameListScreen from '../src/gameList/GameListScreen';
+import { newEmptyGame } from '../src/services/GameService';
+
+import renderer from 'react-test-renderer';
+
+describe('Screens tests', () => {
```

Comment:
I know that's a test we see `create-react-app` and `create-react-native-app` but they don't bring much. You must test whether your components react correctly when manipulated (changing props, etc.)